docker/mojopaste.conf

```diff
@@ -0,0 +1,7 @@
+{
+  paste_dir     => '/usr/src/app/data',
+  enable_charts => 1, # default is 0
```

Comment:
I would rather use environment variables for this, since it's easier to set from inside the Dockerfile:

```
ENV PASTE_ENABLE_CHARTS 1
ENV PASTE_DIR /data/paste
```

Reply:
Will this work for `mojopaste prefork`? Now in Dockerfile already has some ENVs
I thought it  work only for `mojopaste deamon`

Reply:
`ENV` will work for any of the servers: morbo, daemon, prefork, ...